tests/conftest.py

```diff
@@ -0,0 +1,136 @@
+"""Set up common pytest fixtures."""
+
+from datetime import datetime
+
+import numpy as np
+import pytest
+from harmony_service_lib.util import bbox_to_geometry
+from pystac import Asset, Catalog, Item
+from xarray import DataArray, DataTree, open_datatree
+
+
+# Fixtures
+@pytest.fixture
+def sample_datatree_file(tmp_path) -> str:
+    """Create a sample DataTree file for testing.
+
+    A sample DataTree is created
+
+    The test data is repeated for both global and polar nodes.
+
+    The tree is written to disk and the filename is returned.
+    """
+    dt = DataTree()
+    dt['Metadata/Lineage/DEMSLP'] = DataTree()
+    dt['Metadata/Lineage/DEMSLP'].attrs[
+        'Description'
+    ] = 'Representative surface slope data for each of the 9 km cells'
+
+    nodes = ['Soil_Moisture_Retrieval_Data', 'Soil_Moisture_Retrieval_Data_Polar']
+    for node in nodes:
+        dt[f'{node}'] = DataTree()
+        dt[f'{node}/EASE_column_index'] = DataArray(
+            data=np.array([1175, 1175, 1175, 1175, 1175], dtype=np.uint16),
+            dims=['phony_dim_0'],
+            name='EASE_column_index',
+            attrs={
+                'long_name': 'The column index of the 9 km EASE grid cell...',
+                'valid_min': 0,
+                'valid_max': 3855,
+                '_FillValue': np.uint16(65534),
+            },
+        )
+
+        dt[f'{node}/EASE_row_index'] = DataArray(
+            data=np.array([1603, 1604, 1605, 1606, 1607], dtype=np.uint16),
+            dims=['phony_dim_0'],
+            attrs={
+                'long_name': 'The row index of the 9 km EASE grid cell...',
+                'valid_min': 0,
+                'valid_max': 1623,
+                '_FillValue': np.uint16(65534),
+            },
+        )
+
+        dt[f'{node}/albedo'] = DataArray(
+            data=np.array(
+                [0.0009434, 0.00136986, 0.0025, 0.0, -9999.0], dtype=np.float32
+            ),
+            dims=['phony_dim_0'],
+            attrs={
+                'long_name': 'Diffuse reflecting power of the Earth&apos;s...',
+                'valid_min': 0.0,
+                'valid_max': 1.0,
+                '_FillValue': np.float32(-9999.0),
+            },
+        )
+
+        # This part of the fixture REALLY slow when running tests.  Using it
+        # adds 7 seconds(!!) to the run time. Really there is only one useful
+        # test that needs this.  I tested this function outside of pytest and
+        # it's not acting like a bottle neck so I'm not sure what to do. But
+        # for now I'll remove this and use mark to skip the test.
```

Comment:
I'm happy to talk through this, but I couldn't understand or work around this slowness.